app/application.js

```diff
@@ -1,18 +1,53 @@
 import Mn from 'backbone.marionette';
 import Backbone from 'backbone';
-import tracksView from 'views/tracks';
+import Tracks from './collections/tracks';
+import Playlists from './collections/playlists';
+import AppLayout from './views/app_layout';
 
+require('./styles/app.styl');
 
-class Application extends Mn.Application {
+const Application = Mn.Application.extend({
 
-    onStart () {
-        console.log('start');
+    initialize: function () {
+        this.allTracks = new Tracks([], { type: 'all' });
+        this.upNext = new Tracks();
+        this.upNext.on('change:currentTrack', function() {
```

Comment:
Why putting in `application` some code dedicated to maintain the internal state of the upNext tracks list ? It'll probably better to keep it inside the `Tracks` module, and let configure it at construct by passing a flag `upNext: true`, nope?